config/manager/controller_manager_config.yaml

```diff
@@ -9,3 +9,77 @@ webhook:
 leaderElection:
   leaderElect: true
   resourceName: e29e094a.k8s.io
+manager:
+  runtime: containerd
+  otlpEndpoint: ""
+  logLevel: info
+  scheduling:
+    repeatInterval: 24h # to be parsed into time.Duration
+    beginImmediately: true
+  profile:
+    enable: false
+    port: 6060
+  imageJob:
+    successRatio: 1.0 # float; ok with YAML?
+    cleanup:
+      delayOnSuccess: 0s # to be parsed into time.Duration
+      delayOnFailure: 24h
+  pullSecrets: [] # image pull secrets for collector/scanner/eraser
+  nodeFilter:
+    type: exclude # must be either exclude|include
+    selectors:
+      - eraser.sh/cleanup.filter
+components:
+  collector:
+    enable: true
+    image:
+      repo: ghcr.io/azure/collector
+      tag: v1.0.0-beta.3
```

Comment:
are these tags going to be automatically updated when we cut a release?

Reply:
This comment no longer applies, see the changes in the Makefile and `build/tooling/Dockerfile.manifest`. Basically placeholder values are there that get overwritten, similar to the way we used to have `kustomize.template.yaml` and sed the right values in.

Reply:
done (see new Dockerfiles)